### ISSUE TITLE
Update to twin.macro

### ISSUE THREAD

**Issue by ben-rogerson** [jlengstorf/gatsby-tailwind-demo#11]
Hey Jason!

I understand this repo is an absolute grandpa in developer years but I found it on Google and thought I could lend a hand with updating it to use my updated fork of tailwind.macro - [twin.macro](https://github.com/ben-rogerson/twin.macro).

Let me know if you're keen and I'll submit a PR.

Cheers!

**Comment by jlengstorf** [comment]
hey, I don't do much with Tailwind, so I'm probably not the right person to have a Gatsby/Tailwind example

if you want to fork and/or create your own example, I'd be happy to direct people there instead of here!

I think I'm going to archive this repo to make it clear that it's outdated and probably shouldn't be used 😅

**Comment by ben-rogerson** [comment]
Yeah no problems, I think you're best off with the archive on this one 👍
